compiler/example/output/generated/crds/gns_gns.yaml

```diff
@@ -269,6 +276,7 @@ spec:
             required:
             - domain
             - useSharedGateway
+            - Annotations
```

Comment:
AFAIK, the field names start with small letter in CRD. Is there any difference here?

Reply:
We have created JIRA: https://jira.eng.vmware.com/browse/COSMOS-3251 to debug this later
This is not impacting